### ISSUE TITLE
Digits (only) keyboard for numeric inputs

### ISSUE THREAD

**Issue by knoppixmeister** [EdgeTX/edgetx#3722]
### Is there an existing issue for this feature request?

- [X] I have searched the existing issues

### Is your feature request related to a problem?

It would be nice to have digits (only) keyboard for numeric input fields like it made e.g. in Android.

example:

<img width="350" alt="kbd_2" src="https://github.com/EdgeTX/edgetx/assets/567882/b67bf23c-765f-4f46-bb7f-fdef392cea03">

### Describe the solution you'd like

Create digits (only) keyboard.

**Comment by ggoraa** [comment]
You know what, this is a great suggestion! I can’t think of a reason to _not_ implement this keyboard layout variant

**Comment by Sea-Kerman** [comment]
yeah I was going to suggest this as well. Probably also want a - sign

**Comment by J-Sorenson** [comment]
...and a decimal point.

**Comment by philmoz** [comment]
The complexity is that the numeric input fields all have range limits (min & max values).
This is difficult to manage / enforce if users can enter arbitrary values.

**Comment by Sea-Kerman** [comment]
just clamp the value

**Comment by pfeerick** [comment]
The question is probably rather one of if we can clamp it at time of input, or if we only get that input after having exited the keyboard... making it somewhat intuitive... (I entered 150, but now the field is only showing 100!!)

**Comment by onliner10** [comment]
I opened a draft PR for this: https://github.com/EdgeTX/edgetx/pull/7338

It replaces the color LCD touch step-control pad with an adaptive full-screen numeric keypad:

- digits are always available
- `+/-` appears only for negative-capable fields
- `.` appears only for `PREC1` / `PREC2` fields
- custom-display fields keep the existing adjustment pad unless they explicitly opt into direct numeric input
- typed values still commit through `NumberEdit::setValue()`, so existing min/max clamping remains authoritative

The PR includes screenshots, notes on the implementation, and the firmware build fix needed for `-fno-rtti`. I also tested it on a TX16S MK2 and verified the fork TX16S firmware workflow builds successfully with strict compiler checks.
